docs/design/2021-04-26-lock-view.md

```diff
@@ -151,9 +155,23 @@ The locking key and `resource_group_tag` that comes from the `Context` of the pe
 
 The wait chain will be added to the `Deadlock` error which is returned by the `PessimisticLock` request, so that when deadlock happens, the full wait chain information can be passed to TiDB.
 
+### Configurations
```

Comment:
Maybe some background HTTP API could be used, currently, we need to be careful adding system variables as there are already many and once released we can't take them back.